libmattermost.c

```diff
@@ -3526,6 +3584,9 @@ mm_add_account_options(GList *account_options)
 	
 	option = purple_account_option_bool_new(N_("Use SSL/HTTPS"), "use-ssl", TRUE);
 	account_options = g_list_append(account_options, option);
+
+	option = purple_account_option_bool_new(N_("Use password as MMAUTHTOKEN"), "use-mmauthtoken", FALSE);

```

Comment:
OK, changed 'MMAUTHTOKEN' to 'Gitlab cookie' in GUI checkbox/errors as well as in docs.